src/linear_algebra/basis.lean

```diff
@@ -65,8 +65,10 @@ noncomputable theory
 open function set submodule
 open_locale classical big_operators
 
+universes v
```

Comment:
Do you really need to call the universe `v` when the first variable definition is also `v`, and `exists_sum_is_basis` will thus involve two orthogonal `v`s? I guess `u` is already taken somehwere in the file, maybe `u'` then?

Reply:
Good point.